src/core/webidl.js

```diff
@@ -672,13 +677,11 @@ function writeMethod(meth, max, indent) {
     });
   var params = paramObjs.join(", ");
   var len = idlType2Text(meth.idlType).length;
-  if (meth.static) len += 7;
   var specialProps = [
     "getter",
     "setter",
     "deleter",
-    "legacycaller",
-    "stringifier",
```

Comment:
Shouldn't we keep `stringifier`? `legacycaller` is removed but `stringifier` is not.

Reply:
Whoops!

Reply:
yeah, will add a test!